var/spack/repos/builtin/packages/py-requests/package.py

```diff
@@ -25,11 +26,15 @@ class PyRequests(PythonPackage):
     variant('socks', default=False, description='SOCKS and HTTP proxy support')
 
     depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
+    depends_on('python@2.7:2.8,3.6:', type=('build', 'run'), when='@2.26.0:')
     depends_on('py-setuptools', type='build')
 
     depends_on('py-chardet@3.0.2:4', type=('build', 'run'), when='@2.25.1:')
     depends_on('py-chardet@3.0.2:3', type=('build', 'run'), when='@2.23.0:2.25.0')
     depends_on('py-chardet@3.0.2:3.0', type=('build', 'run'), when='@2.16.0:2.22')
+    depends_on('py-charset-normalizer@2.0.0:2', type=('build', 'run'), when='@2.26.0: ^python@3:')
+    depends_on('py-idna@2.5:3', type=('build', 'run'), when='@2.26.0: ^python@3:')
+    depends_on('py-idna@2.5:2', type=('build', 'run'), when='@2.26.0: ^python@2:2')
     depends_on('py-idna@2.5:2', type=('build', 'run'), when='@2.23.0:')
```

Comment:
```suggestion
    depends_on('py-idna@2.5:2', type=('build', 'run'), when='@2.23:2.25')
```
Otherwise this constraint will be applied to all versions